src/mscorlib/src/System/Buffer.cs

```diff
@@ -507,8 +507,9 @@ internal unsafe static void Memmove(byte* dest, byte* src, nuint len)
             *(nuint*)dest = *(nuint*)src;
 
             // byte offset at which we're copying
-            int offset = sizeof(nuint) - ((int)dest % sizeof(nuint)); // maps 0 => 8, 1 => 7, ... 8 => 8 for 64-bit
```

Comment:
@jamesqo, why do negatives need to be handled? You should be able to do `uint offset = sizeof(nuint) - ((uint)(dest) % sizeof(nuint))`